app/services/data/get-arms-totals.js

```diff
@@ -0,0 +1,36 @@
+const config = require('../../../config')
+const knexConfig = require('../../../knexfile').app
+const knex = require('knex')(knexConfig)
+
+module.exports = function (omKey, teamCode) {
+  var whereObject = {
+    'assessment_staff_key': omKey,
+    'assessmentent_team_key': teamCode // typo in extract
+  }
+
+  return knex.count('sentence_type as count').select('sentence_type')
+          .from(`${config.DB_STG_SCHEMA}.arms`)
+          .where(whereObject) // .andWhere(disposal_or_release_date is no more than 6 weeks ago)
```

Comment:
Where is this `.andWhere` set?

Reply:
This is an overhang, I've confirmed with Kristy that we'll only get ARMs assessments from the last 6 weeks so I'll remove this comment, cheers.